### PR TITLE
Blog: Bitcoin Grants 6th Wave

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build preview:**
- [/blog/bitcoin-grants-july-2024-6th-wave](https://os-website-git-bitcoin-grants-6th-wave-open-sats.vercel.app/blog/bitcoin-grants-july-2024-6th-wave)

**Social preview:**
41-bitcoin-6th-wave

### CODE REVIEW

**Comment by Arvin21M** [comment]
@dergigi Need your help to create the blog header image for this announcement 🙏

**Comment by dergigi** [comment]
Will get on it on Monday 🤙

File: data/blog/bitcoin-grants-6th-wave.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/TODO.jpg']
+summary: "OpenSats is funding 8 more open-source projects in the bitcoin ecosystem."
+---
+
+We are excited to announce the sixth wave of Bitcoin grants, reaffirming our commitment to fostering innovation and development within the open-source community. 
+An important part of our mission is to provide essential funding to developers and projects that advance the Bitcoin ecosystem.
+
+In this latest round, we are proud to support eight remarkable projects that exemplify the spirit of open-source and technological advancement. In no particular order, these projects are:
```
Comment (dergigi) [inline data/blog/bitcoin-grants-6th-wave.mdx:14]:
```suggestion
In this latest round, we are proud to support eight remarkable projects that exemplify the spirit of our mission. In no particular order, these projects are:
```

File: data/blog/bitcoin-grants-6th-wave.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+-   [Bitcoin Jungle](#bitcoin-jungle)
```
Comment (dergigi) [inline data/blog/bitcoin-grants-6th-wave.mdx:16]: I'd suggest we move Bitcoin Jungle to the end of the list. We usually lead with the tech, and end on education.

File: data/blog/bitcoin-grants-6th-wave.mdx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+-   [secp256k1.swift](#secp256k1.swift)
+-   [Blitz Wallet](#blitz-wallet)
+-   [NLightning](#nlightning)
+-   [Bitcoin Fees](#bitcoin-fees)
+-   [eNuts](#enuts)
+-   [Peer Storage Backups for Lightning Channels](#peer-storage-backups-for-lightning-channels)
+-   [Gonuts](#gonuts)
+
+Made possible by the generosity of our donors, these grants enable contributors and projects to continue their critical work and drive the future of Bitcoin forward.
```
Comment (dergigi) [inline data/blog/bitcoin-grants-6th-wave.mdx:25]:
```suggestion
Made possible by the generosity of our donors, these grants enable the various contributors to continue their critical work into the future.
```

File: data/blog/bitcoin-grants-6th-wave.mdx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+Lightning backends beyond LND. These enhancements will make Gonuts more robust, versatile, and user-friendly, fostering broader adoption and innovation within the Cashu ecosystem.
+
+Repository: [elnosh/gonu](https://github.com/elnosh/gonuts)
+
+License: MIT
+
+---
+
+With this 6th wave of grants, much like our prior waves, we evaluated applications based on carefully defined [criteria](/apply#criteria) and did our best to select only the most impactful 
+and promising proposals for the Bitcoin ecosystem. If you are a developer working on a project that is aligned with [our mission](https://opensats.org/mission), don't hesitate to [apply for funding](/apply).
```
Comment (dergigi) [inline data/blog/bitcoin-grants-6th-wave.mdx:142]:
```suggestion
and promising proposals for the Bitcoin ecosystem. If you are a developer working on a project that is aligned with [our mission](/mission), don't hesitate to [apply for funding](/apply).
```

**Review by dergigi — CHANGES_REQUESTED** [review]
Great job! I left some comments and made some minor suggestions.

File: data/blog/bitcoin-grants-6th-wave.mdx (hunk 2 above)
Comment (Arvin21M) [reply data/blog/bitcoin-grants-6th-wave.mdx]: Done 🖖

**Review by Arvin21M — COMMENTED** [review]
Thanks for all the improvements and corrections.

Look great! 🚀

**Review by Arvin21M — COMMENTED** [review]
In case no one else has checked yet.... Links are all good and point to the right place. Could not discover additional punctuation or spelling typos. Good to go 🖖

**Review by dergigi — APPROVED** [review]

**Comment by dergigi** [comment]
It's live!
- https://opensats.org/blog/bitcoin-grants-july-2024-6th-wave
- https://njump.me/nevent1qqs9nn8quv0cp9q6dq087a9tvm6u5dtq5v5djfxdrqjvyczzxl8hftszypu8xwr40lp96ewdj2fef408wy70gd3carf9n6xu7hrnhq6whpglyhzksvx

**Pull request merged**